pkg/config/system_test.go

```diff
@@ -168,7 +169,7 @@ func TestGetLargestID(t *testing.T) {
 		{[]roachpb.KeyValue{
 			sqlKV(keys.NamespaceTableID, 1, 1),
 			sqlKV(keys.NamespaceTableID, 1, 2),
-			sqlKV(keys.UsersTableID, 1, 3),
+			sqlKV(keys.LeaseTableID, 1, 3),
```

Comment:
I just noticed that this change is gratuitous. It also doesn't have an impact on anything. I'll get rid of it at the next rebase. Disregard.

Reply:
Done.